docs/testkit/sample_input/v-t_0003.a_vc_embedded_forged_claim.json

```
@@ -0,0 +1,37 @@
+{
+  "issuanceDate": "2023-06-12T13:13:30Z",
+  "credentialSubject": {
+    "bpn": "BPNL00000000invalid",
+    "id": "did:web:localhost:BPNL000000000000",
+    "type": "BpnCredential"
+  },
+  "id": "7f6c11b4-d2b9-43c3-8411-53f6089b5d2b",
+  "type": [
+    "VerifiableCredential",
+    "BpnCredentialCX"
+  ],
+  "issuer": "did:web:localhost:BPNL000000000000",
+  "expirationDate": "2024-12-31T23:00:00Z",
+  "sec:proof": {
+    "type": "JsonWebSignature2020",
+    "created": "2022-12-31T23:00:00Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": {
+      "type": "JsonWebKey2020",
+      "publicKeyJwk": {
+        "kty": "EC",
+        "crv": "P-384",
+        "x": "eQbMauiHc9HuiqXT894gW5XTCrOpeY8cjLXAckfRtdVBLzVHKaiXAAxBFeVrSB75",
+        "y": "YOjxhMkdH9QnNmGCGuGXJrjAtk8CQ1kTmEEi9cg2R9ge-zh8SFT1Xu6awoUjK5Bv"
+      },
+      "id": "https://org.eclipse.tractusx/keys/68c7189c-b849-4f85-b27d-c796c7cf29ed"
+    },
+    "jws": "eyJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCJdLCJhbGciOiJFUzM4NCJ9..0ueANOomarONwEL2Y0QnCFjgOdgPjI8kL2Wk4QWh8SJjvVTR80ASVh7bi8HlQp6dUigP3r509oMQkXB6TEddi0D8oQc2Lv0uWxl7yxPInBcfIsWmQrFBTb4mCSU_MJwE"
+  },
+  "@context": [
+    "https://org.eclipse.tractusx/businessPartnerData",
```

```suggestion
    "https://org.eclipse.tractusx/businessPartnerData.json",
```
Because of the github pages issue it should be with .json I think